### ISSUE TITLE
Gallery: Some drawer item actions are not accessible

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#14471]
In the gallery, when the following items from the drawer are selected there is no feedback for screen reader users about the result of the action:

* Animate Slowly
* Checkerboard Offscreen Layers
* Checkerboard Raster Cache Images
* Performance Overlay

(This was discovered in an a11y review)

**Comment by Hixie** [comment]
Don't we announce that the checkbox is now checked, in each case?

**Comment by goderbauer** [comment]
You're right. That change was probably not included in the gallery version that was used for the review.
